Makefile

```diff
@@ -140,8 +149,7 @@ sync-external-ip:
 
 .PHONY: kubeadm-join
 kubeadm-join:
-	$(NODE_SHELL) sh -euc "envsubst </usernetes/kubeadm-config.yaml >/tmp/kubeadm-config.yaml"
```

Comment:
Two notes here:

1. I made the `join-command` executed with bash so it will work in a read only filesystem
2. I wasn't sure if we needed the first command if the node isn't a control plane, because it seemed to generate that kubeadm-config.yaml in /tmp but then not do anything with it. Let me know if we do and I'll add back the line (and I'm curious to know how it's used).